### DISCUSSION TITLE
problems in parasitic capacitance extraction

### DISCUSSION THREAD

**Discussion by DTforwhat** [The-OpenROAD-Project/OpenROAD#6266] [Q&A]
I noticed there's a rc file in test/Nangate45/Nangate45.rc  I wonder how these Cap figures been calculated

**Comment by maliberty** [comment]
@pradi001 set this up a long time ago.  As there is no fab the numbers are somewhat arbitrary.

**Reply by DTforwhat** [reply]
thx 4 reply!
I wonder what's the relationship between parasitic capacitance and the capacitance setting in this file
Does it get involved in parasitic capacitance extraction?
